### PR TITLE
[DSCP-418] Deploy Validator service

### PR DESCRIPTION
**Pull request by mkaito**
Along with some background fixes, the validator service is now running:

http://validator.faircv.dscp.serokell.review/

### CODE REVIEW

**Comment by mkaito** [comment]
I could really have done a better job splitting the non-specific parts of this PR. Feel free to review anyway, but bear in mind that it's kind of dirty.

File: shell.nix [hunk 1]
```diff
@@ -14,5 +14,6 @@ stdenv.mkDerivation {
     "disciplina=https://github.com/DisciplinaOU/disciplina/archive/master.tar.gz"
     "disciplina-faucet-frontend=https://github.com/DisciplinaOU/disciplina-faucet-frontend/archive/master.tar.gz"
     "disciplina-explorer-frontend=https://github.com/DisciplinaOU/disciplina-explorer-frontend/archive/master.tar.gz"
+    "disciplina-validatorcv=https://github.com/DisciplinaOU/disciplina-validatorcv/archive/release/0.1.0.tar.gz"
```
Comment (truekubas) [inline shell.nix]: let me merge release into master though? i think there wont be any fixes today on frontend

Comment (mkaito) [reply shell.nix]: Sure. When you do that, feel free to send a PR here as well.

**Comment by mkaito** [comment]
It's basically the same as explorer and faucet. Static front that talks to a backend API. Except this one is React.

**Pull request merged**